src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd

```diff
@@ -368,7 +368,7 @@
     <xsd:complexType name="exception">
         <xsd:attribute name="name" type="xsd:string" use="required" />
         <xsd:attribute name="log-level" type="xsd:string" />
-        <xsd:attribute name="status-code" type="xsd:int" />
+        <xsd:attribute name="status-code" type="xsd:string" />
```

Comment:
Probably there may be another way to allow specifying `null` for XML attribute, but I couldn't find any.

Does someone know a more proper way?

Reply:
Not a problem anymore since https://github.com/symfony/symfony/pull/46956#discussion_r982537440